crypto/keyring/keyring_test.go

```diff
@@ -1161,4 +1227,23 @@ func requireEqualInfo(t *testing.T, key Info, mnemonic Info) {
 	require.Equal(t, key.GetType(), mnemonic.GetType())
 }
 
+func newKeyring(t *testing.T, name string) Keyring {
+	kr, err := New(name, "test", t.TempDir(), nil)
+	require.NoError(t, err)
+	return kr
+}
```

Comment:
This function is also only called once and, given it contains only a few lines of code, it might not be necessary to extract but I think it's ok as is.

Reply:
i'm not sure if it actually behaves this way, but since it's inside the for loop, my intention was to have the keyring clear each test case, to avoid collisions. even though there technically aren't with my test cases, i felt it could be helpful if more cases were to be added later on 🤷🏻